src/Components/ErrorPages/BrowserWarning.tsx

```diff
@@ -0,0 +1,42 @@
+import  { useEffect, useState } from "react";
+import browserslist from "browserslist";
+import { getUserAgentRegex } from "browserslist-useragent-regexp";
+import packageJson from "../../../package.json";
+
+const BrowserWarning = () => {
+  const [isSupported, setIsSupported] = useState<boolean>(true);
+
+  useEffect(() => {
+    const userAgent = window.navigator.userAgent;
+    const supportedBrowsers = browserslist(packageJson.browserslist.production);
+    console.log("Supported Browsers List:", supportedBrowsers);
+
+    const regex = getUserAgentRegex({
+      browsers: supportedBrowsers,
+      allowHigherVersions: true,
+      ignorePatch: true,
+      ignoreMinor: true,
+    });
+
+    if (!regex.test(userAgent)) {
+      setIsSupported(false);
+    }
+  }, []);
+
+  if (isSupported) {
+    return null;
+  }
+
+  return (
+    <div className="fixed left-0 top-0 z-50 w-full h-20 bg-gray-800 bg-opacity-60 flex items-center justify-center text-center text-gray-300">
+    <div>
+      <h2 className="text-lg font-medium">Unsupported Browser</h2>
+      <p className="text-sm">
+        You are using an unsupported browser. Please switch to a supported browser for the best experience.
+      </p>
+    </div>
```

Comment:
@AdityaJ2305 can we improve the prompt message?

This does not guide the user to take an action properly. 

Could we specify something "update the browser version" so that the user knows what to be done?

Reply:
<img width="1440" alt="Screenshot 2024-08-26 at 1 59 58 PM" src="https://github.com/user-attachments/assets/12f5dfa7-045b-4eb6-8525-87a9a3670b35">
Does it look good ? I need to install a library called bower to extract the exact info regarding browser we can do it without installing bower library with window.navigator.userAgent but it works only for a few browsers properly. to get the wide of browsers detail I need to use the bower library 
Let me know if any changes to be made I am not commiting it without any confirmation